habitat-lab/habitat/tasks/rearrange/sub_tasks/cat_nav_to_obj_sensors.py

```diff
@@ -86,8 +87,9 @@ def _get_obs_channel(self, pan_obs, max_obs_val, goals, goals_type):
                 ].parent_object_handle
                 obj_id = rom.get_object_id_by_handle(handle)
             instance_id = obj_id + self._instance_ids_start
-            # Skip if object is not in the agent's viewport
-            if instance_id > max_obs_val:
+            # Skip if receptacle is not in the agent's viewport or if the instance
+            # is selected to be blanked out randomly
+            if instance_id > max_obs_val or np.random.random() < self._blank_out_prob:
```

Comment:
Need to add `_blank_out_prob` in the configs

Reply:
Missed pushing the change